src/App.vue

```diff
@@ -317,6 +338,62 @@ utrecht-button.button-small {
   --utrecht-button-padding-block-end: 0.6rem;
 }
 
+.utrecht-button {
+  background-color: var(--utrecht-button-background-color);
+
+  border-color: var(
+    --utrecht-button-border-bottom-color,
+    var(--utrecht-button-border-color, transparent)
+  );
+
+  border-width: var(
+    --utrecht-button-border-bottom-width,
+    var(--utrecht-button-border-width, 0)
+  );
+  color: var(--utrecht-button-color);
+  font-family: var(
+    --utrecht-button-font-family,
+    var(--utrecht-document-font-family)
+  );
+  font-size: var(
+    --utrecht-button-font-size,
+    var(--utrecht-document-font-family)
+  );
+  font-weight: var(--utrecht-button-font-weight);
```

Comment:
zitten al deze regels niet al ergens in een css bestand?

Reply:
klopt er miste ergens een import. verwijderd